package.json

```diff
@@ -2,6 +2,7 @@
   "name": "tape-jsx-equals",
   "version": "1.0.0",
   "description": "tape assertion to compare react components",
+  "repository": "https://github.com/atabel/tape-jsx-equals.git"
```

Comment:
Maybe you can just close this PR and add it yourself? Faster than me cloning + force pushing a single commit (made from github in the first place :))